service idfServer{    
    double get_word_idf(1:string word, 2:string language),    
}    
